pkg/windowsdesktop/sfx/Microsoft.WindowsDesktop.App.Ref.sfxproj

```diff
@@ -72,4 +73,7 @@
   <!-- Windows Forms validation and packaging -->
   <Import Project="WindowsForms.Packaging.targets" />
 
+  <ItemGroup Condition="'$(CreatePlatformManifest)' == 'false'">
+    <FilesToPackage  Include="PlatformManifest.txt" TargetPath="data" GeneratedBuildFile="true" />
```

Comment:
Do we need to checkin the actual frozen file?

Reply:
🤦 yep